torch/testing/_internal/common_device_type.py

```diff
@@ -724,6 +725,8 @@ def get_device_type_test_bases():
         test_bases.append(CPUTestBase)
         if torch.cuda.is_available():
             test_bases.append(CUDATestBase)
+        if torch.xpu.is_available():
```

Comment:
Since XPU have not enable all the test case, please don't add this,  the failed XPU CI is caused by this.  use allow_xpu=True is enough:
eg:
```
instantiate_device_type_tests(
    TestProfiler,
    globals(),
    only_for=DEVICE_LIST_SUPPORT_PROFILING_TEST,
    allow_xpu=ALLOW_XPU_PROFILING_TEST,
)
```